src/main/java/com/volunteer/enterprise/VolunteerAppController.java

```diff
@@ -1,51 +1,73 @@
 package com.volunteer.enterprise;
-import dto.Volunteer;
+import com.volunteer.enterprise.dto.Volunteer;
+import com.volunteer.enterprise.service.IVolunteerService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
 
+import java.util.List;
+
 @Controller
 public class VolunteerAppController {
+    IVolunteerService volunteerService;
 
- @Autowired
- VolunteerService volunteerService;
```

Comment:
The change to interface type is good, but why remove the @Autowired annotation?   Not having that removes Spring Boot from object instantiation.